src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -7,40 +7,64 @@ category: adapter
 i18nReady: true
 ---
 
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
 An SSR adapter for use with Cloudflare Pages Functions targets. Write your code in Astro/Javascript and deploy to Cloudflare Pages.
 
 ## Install
```

Comment:
Maybe a more general question, does it make sense to have sub-headings (Auto, Manual)?

Reply:
There is some inconsistency, for sure. It would be a good idea to make another issue and PR after this to standardize the `install` step. Some have the `astro add command` header, and some have Auto/Manual subheaders.

Reply:
> There is some inconsistency, for sure. It would be a good idea to make another issue and PR after this to standardize the `install` step. Some have the `astro add command` header, and some have Auto/Manual subheaders.

@sarah11918 is standardizing the install instructions something you are interested in, or should we just leave them as they are?

Reply:
@VoxelMC I'm in favour of standardizing! Would also be OK for that to happen in this PR, to save the translators from multiple updates!

Reply:
@sarah11918 

The ones in question are: 
- `Lit`, `React`, `Solidjs`, `Svelte`, `Vue` (All have `astro add` command), 
- `Cloudflare` (slightly different, no subheadings, aforementioned), 
- `Netlify`, `Node`, `Vercel` (has manual section, quick install is missing).

If you want, I can make a PR into `update-integrations` because I don't think I can straight-up edit this branch.

I think the best way would be to have headings like this:

```md
## Installation
### Quick Install
### Manual Install
### `other`
```

Reply:
@VoxelMC - Yes, I will clean up the rest of the edits/file (while presumably you're still asleep) and let you make a PR to this PR/branch specifically to unify the headings as appropriate.